corehq/apps/cleanup/models.py

```diff
@@ -25,3 +25,13 @@ class Meta:
             models.UniqueConstraint(fields=['doc_id', 'object_class_path'],
                                     name='deletedsqldoc_unique_id_and_type')
         ]
+
+
+def create_deleted_sql_doc(doc_id, class_path, domain, deleted_on, deleted_by=None):
```

Comment:
I think `deleted_by` should be required. If there are system actions with no obvious username associated maybe they can use a name like 'system'?

Reply:
Currently, we don't keep track of who deleted or even last modified forms and cases (there is a last `modified_by` field for cases but that's only updated on followup form submissions) which means all deletions will basically have the same `deleted_by`. I included the column there to encourage future work so we can either add that field or for cases, extend the usage of `modified_by`. Also, I don't think we need to create tombstones for deleted system forms.